app/services/application/ApplicationService.scala

```diff
@@ -151,13 +151,24 @@ trait ApplicationService extends EventSink {
       appRepository.findByUserId(userId, frameworkId).flatMap { appResponse =>
         (appResponse.progressResponse.fastPassAccepted, appResponse.applicationRoute) match {
           case (true, _) => schemeRepository.find(appResponse.applicationId).map(_.schemes)
+
           case (_, ApplicationRoute.Edip | ApplicationRoute.Sdip) =>
             evaluateP1ResultService.getPassmarkEvaluation(appResponse.applicationId).map(passedSchemes)
+
+          case (_, ApplicationRoute.SdipFaststream) => getSdipFaststreamSchemes(appResponse.applicationId)
+
           case _ => evaluateP3ResultService.getPassmarkEvaluation(appResponse.applicationId).map(passedSchemes)
         }
       }
   }
 
+  private def getSdipFaststreamSchemes(applicationId: String): Future[List[SchemeType]] = for {
+    phase1 <- evaluateP1ResultService.getPassmarkEvaluation(applicationId)
+    phase3 <- evaluateP3ResultService.getPassmarkEvaluation(applicationId)
```

Comment:
phase3 wont be available if SdipFaststream candidate failed phase2/phase3 but passed in phase1